cv-data-service-library/src/main/java/com/trihydro/library/helpers/CdotGisConnector.java

```diff
@@ -0,0 +1,45 @@
+package com.trihydro.library.helpers;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+
+import com.trihydro.library.service.RestTemplateProvider;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestClientException;
+
+@Component
+public class CdotGisConnector {
+  private final String baseUrl = "https://dtdapps.coloradodot.info/arcgis/rest/services/LRS/Routes/MapServer/exts/CdotLrsAccessRounded";
+
+  private final RestTemplateProvider restTemplateProvider;
+
+  private final Logger logger = LoggerFactory.getLogger(CdotGisConnector.class);
+
+  @Autowired
+  public CdotGisConnector(RestTemplateProvider _restTemplateProvider) {
+    this.restTemplateProvider = _restTemplateProvider;
+  }
+
+  public String getBaseUrl() {
+    return baseUrl;
+  }
+
+  public RestTemplateProvider getRestTemplateProvider() {
+    return restTemplateProvider;
+  }
+
+  public ResponseEntity<String> getRouteById(String routeId) throws RestClientException {
```

Comment:
What does this return type look like? Should we add some comments to this method to show what we're returning?

Reply:
Added JavaDocs to clarify that the return type is in JSON format and contains all latitude/longitude points in the route.